realityflux/chains.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "../chains.schema.json",
+  "name": "Reality Flux",
+  "chains": [
+    {
+      "name": "evmos",
+      "address": "evmosvaloper1n4zajqylnmk7kmnymq4ceynzlkq68j9l6qrxmg",
+      "restake": {
+        "address": "evmos152r7uy6scz9kpfxcjuy3frg4vhqeawfnlqtzhr",
+        "run_time": "once a day",
```

Comment:
This will need to be changed to the specific runtime, e.g "run_time": "09:00". Time is in UTC.

Reply:
Okay, it's updated now.